diff --git a/src/android/app/build.gradle.kts b/src/android/app/build.gradle.kts
index 185704c8fe..94e2d503c3 100644
--- a/src/android/app/build.gradle.kts
+++ b/src/android/app/build.gradle.kts
@@ -60,7 +60,7 @@ android {
     }
 
     defaultConfig {
-        applicationId = "dev.eden.eden_emulator"
+        applicationId = "dev.eden.eden_nightly"
         minSdk = 24
         targetSdk = 36
         versionName = getGitVersion()
@@ -174,8 +174,8 @@ android {
             dimension = "version"
             isDefault = true
 
-            manifestPlaceholders += mapOf("appNameBase" to "Eden")
-            resValue("string", "app_name_suffixed", "Eden")
+            manifestPlaceholders += mapOf("appNameBase" to "Eden Nightly")
+            resValue("string", "app_name_suffixed", "Eden Nightly")
 
             ndk {
                 abiFilters += listOf("arm64-v8a")
