diff --git a/src/video_core/renderer_opengl/gl_texture_cache.cpp b/src/video_core/renderer_opengl/gl_texture_cache.cpp
index 14ab5dd967..958988f27e 100644
--- a/src/video_core/renderer_opengl/gl_texture_cache.cpp
+++ b/src/video_core/renderer_opengl/gl_texture_cache.cpp
@@ -1219,7 +1219,7 @@ GLuint ImageView::StorageView(Shader::TextureType texture_type, Shader::ImageFor
     const bool is_signed = image_format == Shader::ImageFormat::R8_SINT
         || image_format == Shader::ImageFormat::R16_SINT;
     if (!storage_views)
-        storage_views.emplace();
+        storage_views = {OpenGL::ImageView::StorageViews{}};
     auto& type_views{is_signed ? storage_views->signeds : storage_views->unsigneds};
     GLuint& view{type_views[size_t(texture_type)]};
     if (view == 0)
